### ISSUE TITLE
[FVT] xcattest - Incorrectly specifying a testcase name does not tell me what went wrong

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#4189]
Specifying a incorrect testcase name to `xcattest` does not have any checking. 

The testcase name was changed from `rflash_list` to `rflash_check` and when I ran it with the old name, the error message was not informative.

```
******************************
loading test cases
******************************
To run:
rflash_list
******************************
Start to run test cases
******************************
Use of uninitialized value in array element at /opt/xcat/bin/xcattest line 1170.
Use of uninitialized value in concatenation (.) or string at /opt/xcat/bin/xcattest line 1170.
------START::::Time:Thu Oct 26 14:53:41 2017------

Use of uninitialized value in array element at /opt/xcat/bin/xcattest line 1171.
Use of uninitialized value in concatenation (.) or string at /opt/xcat/bin/xcattest line 1171.
Use of uninitialized value in array element at /opt/xcat/bin/xcattest line 1172.
Use of uninitialized value in concatenation (.) or string at /opt/xcat/bin/xcattest line 1172.
Use of uninitialized value in array element at /opt/xcat/bin/xcattest line 1174.
Use of uninitialized value in array element at /opt/xcat/bin/xcattest line 1307.
Use of uninitialized value in concatenation (.) or string at /opt/xcat/bin/xcattest line 1307.
------END::::Passed::Time:Thu Oct 26 14:53:41 2017 ::Duration::0 sec------
Use of uninitialized value in array element at /opt/xcat/bin/xcattest line 1308.
Use of uninitialized value in concatenation (.) or string at /opt/xcat/bin/xcattest line 1308.
------Total: 1 , Failed: 0------

xCAT automated test finished atThu Oct 26 14:53:41 2017
Please check results in the /opt/xcat/bin/../share/xcat/tools/autotest/result/, 
and see /opt/xcat/bin/../share/xcat/tools/autotest/result//failedcases.20171026145339 file for failed cases.
see /opt/xcat/bin/../share/xcat/tools/autotest/result//performance.report.20171026145339 file for time consumption
```

**Comment by whowutwut** [comment]
To lump in another issue, the date string is missing a space: 

`xCAT automated test finished atThu Oct 26 15:27:25 2017`

**Comment by hu-weihua** [comment]
@whowutwut, thank you for finding the issue, I will fix it in next sprint. thanks

**Comment by hu-weihua** [comment]
@whowutwut , this issue has been fixed by PR #4279. Could you please verify it? You also can refer to the description of #4279 to get the output of fix. thanks.

**Comment by hu-weihua** [comment]
@whowutwut . I have not received any feedback for this issue up to now, I suppose the fix has worked. Close this issue first, if you find any problem in future, you can reopen this issue again, thanks.
